src/main/java/de/tum/in/www1/artemis/web/rest/ExerciseResource.java

```diff
@@ -327,5 +328,43 @@ public ExerciseResource(ExerciseRepository exerciseRepository, ExerciseService e
             .header("filename", zipFile.getName())
             .body(resource);
     }
+    /**
+    * GET  /exercises/:id/participations/:studentIds : sends all submissions from studentlist as zip
+        *
+        * @param id the id of the exercise to get the repos from
+        * @param studentIds the studentIds seperated via semicolon to get their submissions
+     * @return ResponseEntity with status
+     */
+    @GetMapping(value = "/exercises/{id}/participations/{studentIds}")
```

Comment:
What happens, if I have any whitespaces in the studentIds? Can it still be represented as URL?

Reply:
the front end actually filters them out. In addition I also added this replacement to the server in case of malformed requests. In addition I also tested positively that all involved method calls handle them correctly.